apps/web/tests/accessibility/axe-integration.test.ts

```
@@ -0,0 +1,952 @@
+      compliantComponents: reports.filter(r => r.violationCount === 0).length,
+      complianceRate: reports.filter(r => r.violationCount === 0).length / reports.length,
+      reports
+    };
+  }
+};
+
+// Export for use in other test files
+export {
+  healthcareAxeConfig,
+  healthcareTestContexts
+};
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)